src/pages/UserPage/UserPage.tsx

```diff
@@ -91,7 +97,44 @@ const UserPage: FC = () => {
     },
   ]
 
-  const onSubmit: SubmitHandler<FormValues> = useCallback(
+  const roleOptions = compact(
+    map(existingRoles, ({ name, id }) => {
+      if (name) {
+        return {
+          label: name,
+          value: name,
+          id: id,
```

Comment:
**[blocker]**

I don't think we need this id field. "value" field is meant for exactly this purpose. Basically for dropdowns "value" is what we need from code side, in this case what we need for sending information to BE and "label" is only for display purposes. Currently you are using "id" instead of "value" and essentially not using "value" at all